Source/Krypton Components/Krypton.Toolkit/Designers/Action Lists/KryptonTextBoxActionList.cs

```diff
@@ -40,13 +40,13 @@ public KryptonTextBoxActionList(KryptonTextBoxDesigner owner)
         /// <value>The Krypton Context Menu.</value>
         public KryptonContextMenu? KryptonContextMenu
         {
-            get => _textBox.KryptonContextMenu;
+            get => _textBox?.KryptonContextMenu;
 
             set
             {
-                if (_textBox.KryptonContextMenu != value)
+                if (_textBox?.KryptonContextMenu != value)
```

Comment:
If text box is null, and value is not then line 51 will fail

Reply:
Reverted